examples/burial-form/BurialAllowance.jsx

```diff
@@ -44,7 +44,7 @@ export default function BurialAllowance(props) {
 
   return (
     <>
-      <Page {...props}>
+      <Page {...props} title="Burial allowance">
```

Comment:
Title should be updated and passed through the route on the index page.

Reply:
Actually the reason for this is it is an internal step of Burial selection. So, the title in progress bar would be "Burial selection" and the title for page would be "Burial allowance" according to the mockups.